docker/dockerfile-python

```
@@ -26,7 +26,7 @@ RUN apt-get update && \
     # lightgbm
     conda install -q -y numpy scipy scikit-learn pandas && \
     git clone --recursive --branch stable --depth 1 https://github.com/Microsoft/LightGBM && \
-    sh ./build-python.sh install && \
+    cd LigthGBM && sh ./build-python.sh install && \
```

```suggestion
    cd ./LightGBM && \
    sh ./build-python.sh install && \
```

Please put this change on a separate line, for consistency with the rest of this Dockerfile (which is done to make it easier to see the sequence of commands).

And please note that it's `LightGBM`, not `LihgtGBM`.